docroot/themes/humsci/humsci_basic/src/scss/utilities/_wysiwyg-text-area.scss

```diff
@@ -268,3 +264,44 @@ a[href*="//"].hs-external-link,
   @include hb-link--inline($icon: true);
   @include hb-external-link-icon;
 }
+
+a.hs-more-link,
+button.hs-more-link,
```

Comment:
When used with button styles, we'll need to update the text and icon colors. The icon is there but you can't see it because it's blending into the background color. When hovering over the button with the more link, the text blends into the background and you can see the icon.
![buttons](https://user-images.githubusercontent.com/12678977/80123132-7a167900-855c-11ea-96b0-ae9eb71b9457.png)

Reply:
They really shouldn't be mixing util classes like this. There is no way to account for all combinations of classes and determining which styles are dominant and which aren't.

Reply:
This did remind me though to make sure that the styles will work when used on a button element so I had to override a couple of defaults.